step.sh

```diff
@@ -84,21 +84,41 @@ echo_details "* api_key: ***"
 echo_details "* build_secret: ***"
 echo_details "* ipa_path: $ipa_path"
 echo_details "* dsym_path: $dsym_path"
+echo_details "* service_info_plist_path: $service_info_plist_path"
 echo_details "* email_list: $email_list"
 echo_details "* group_aliases_list: $group_aliases_list"
 echo_details "* notification: $notification"
 echo_details "* release_notes: $release_notes"
 
 echo
 
-validate_required_input "api_key" $api_key
-validate_required_input "build_secret" $build_secret
-
 if [ -z "${dsym_path}" ] && [ -z "${ipa_path}" ] ; then
 	echo_fail "No ipa_path nor dsym_path defined"
 fi
 
 if [ ! -z "${ipa_path}" ] ; then
+	validate_required_input "api_key" $api_key
```

Comment:
You check the api_key & build_secret twice

here:
https://github.com/bitrise-steplib/steps-fabric-crashlytics-beta-deploy/pull/27/files#diff-11268f778298d21634cd733e189c62dcR100

and here
https://github.com/bitrise-steplib/steps-fabric-crashlytics-beta-deploy/pull/27/files#diff-11268f778298d21634cd733e189c62dcR118

Reply:
fixed